Core/Core/Groups/Group.swift

```diff
@@ -53,6 +53,10 @@ public final class Group: NSManagedObject, WriteableModel {
         return course
     }
 
+    public var isActive: Bool {
+        courseID == nil || getCourse() != nil
```

Comment:
If `getCourse()` returns an instance, would it be possible to check that course's state? If I navigate to `All courses` go back and do a pull-to-refresh the group still appears for a moment.

https://user-images.githubusercontent.com/72396990/128989141-f4c767b6-78cd-4264-bd66-0ad8c161ac29.MP4